subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -120,7 +133,8 @@ def build_lts_payloads() -> dict:
                 "test": "rhods-notebooks-ux",
                 "start": start_time.isoformat(),
                 "end": end_time.isoformat(),
-                "settings": _parse_entry(entry.settings)
+                "settings": _parse_entry(entry.settings),
+                "cluster_info": _parse_entry(entry.results.rhods_cluster_info)
```

Comment:
```suggestion
                "cluster_info": _parse_entry(entry.results.rhods_cluster_info),
```
adding a trailing `,` makes the diff more readable
see this diff? `"settings": _parse_entry(entry.settings)` didn't actually change, but the comma was "missing", so it appears as modified